lib/api/dsl/storage/index.js

```diff
@@ -14,13 +35,13 @@ const
   FieldOperand = require('./objects/fieldOperand'),
   containsOne = require('../util/containsOne');
 
+/**
+ * Real-time engine filters storage
+ *
+ * @class Storage
+ * @param {string} hashKey
+ */
 class Storage {
-
-  /**
-   * Real-time engine filters storage
-   * @constructor
-   * @param {string} hashKey
-   */
   constructor (hashKey) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This function has 119 lines, which is greater than the 100 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS138)